middlewares/channel.js

```diff
@@ -40,4 +40,16 @@ function channelIfActive(req, res, next) {
 	channelIfFind({ _id: req.params.id, 'spec.validators.id': req.whoami }, req, res, next)
 }
 
-module.exports = { channelLoad, channelIfExists, channelIfActive }
+// requires channelLoad
+function channelIfGrace(req, res, next) {
+	const { channel } = req
+	const currentTime = Date.now()
+	if (channel.spec.gracePeriod || currentTime > channel.validUntil) {
```

Comment:
this seems wrong - the value of `channel.spec.gracePeriod > channel.validUntil` will always be constant

should be `currentTime - gracePeriod`